### PR TITLE
Add related name to Comments.author

### PR DESCRIPTION
**Pull request by bernd-wechner**
Fixing Issue 126: 

https://github.com/shacker/django-todo/issues/126

### CODE REVIEW

File: todo/migrations/0012_auto_20210922_2205.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.24 on 2021-09-22 22:05
```
Comment (shacker) [inline todo/migrations/0012_auto_20210922_2205.py:1]: Please give the migration a useful filename, like `0012_set_comments_related_name`

File: todo/models.py [hunk 2]
```diff
@@ -133,7 +133,8 @@ class Comment(models.Model):
     """
 
     author = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, blank=True, null=True
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, blank=True, null=True,
+        related_name="todo_comments"
```
Comment (shacker) [inline todo/models.py:137]: I would think this change would break todo code in at least one place, no? Will test soon. But the system still works for you?

Comment (bernd-wechner) [reply todo/models.py:137]: All existing tests passed.  The code is running live an in use.  And there are no references to comment_set, the default related_name anywhere in the code base.  Looks safe to me.

File: todo/migrations/0012_auto_20210922_2205.py (hunk 1 above)
Comment (bernd-wechner) [reply todo/migrations/0012_auto_20210922_2205.py:1]: Can do.  Have to work out how first 😉

Comment (bernd-wechner) [reply todo/migrations/0012_auto_20210922_2205.py]: Done

File: todo/models.py (hunk 2 above)
Comment (shacker) [reply todo/models.py:137]: Excellent, thank you for looking.

**Review by shacker — APPROVED** [review]
Migrates and works perfectly for me, thanks!

**Pull request merged**